### PR TITLE
Puppet4 is much stricter about arrays and strings, make all strings

### PR DESCRIPTION
**Pull request by dugdigg**
Needed to change some arrays to strings to handle the new and improved Puppet.

### CODE REVIEW

**Comment by adamcstephens** [comment]
This isn't about puppet4, this is about incorrectly documented python module README which says it is an array when it in fact is a manifest:

https://github.com/stankevich/puppet-python/issues/162#issuecomment-166968768

**Comment by everestx** [comment]
Puppet 3.x was converting the arrays to strings (somewhere), so this all worked great before upgrading to puppet 4.

Looks good!

File: manifests/cli/plugin.pp [hunk 1]
```diff
@@ -3,7 +3,7 @@
   $module,
   $repo=None,
   $version='present',
-  $install_args=[],
+  $install_args,
```
Comment (everestx) [inline manifests/cli/plugin.pp:6]: This shouldn't be a required parameter.  Probably should be an `undef`.

**Comment by adamcstephens** [comment]
Opened README PR for this change: https://github.com/stankevich/puppet-python/pull/282

**Pull request merged**